### PR TITLE
4.0.0-beta

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+* define new DNS failure LH Error ([#6579](https://github.com/googlechrome/lighthouse/pull/6579))
+* assert securityIssues before getPageLoadError ([#6578](https://github.com/googlechrome/lighthouse/pull/6578))
+* remove protocol timeout for Page.navigate ([#6413](https://github.com/googlechrome/lighthouse/pull/6413))
+* rename webapp-install-banner audit to installable-manifest ([#6630](https://github.com/googlechrome/lighthouse/pull/6630))
+* computed-artifacts: fix cache and add perf timing ([#6619](https://github.com/googlechrome/lighthouse/pull/6619))
+
+## Report
+
+* fix pwa-optimized badge gauge ([#6670](https://github.com/googlechrome/lighthouse/pull/6670))
+* add PWA category badge gauge ([#6526](https://github.com/googlechrome/lighthouse/pull/6526))
```
Comment (brendankenny) [inline changelog.md]: the "fix" above one could probably be added to this one

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+* give passed-opportunities group an icon ([#6594](https://github.com/googlechrome/lighthouse/pull/6594))
+* safari: fix dropdown overlap translateZ ([#6546](https://github.com/googlechrome/lighthouse/pull/6546))
+* use details-element-polyfill from node_modules ([#6593](https://github.com/googlechrome/lighthouse/pull/6593))
+* import <detail> polyfill as string, not Buffer ([#6607](https://github.com/googlechrome/lighthouse/pull/6607))
```
Comment (brendankenny) [inline changelog.md]: these two could be grouped, too

**Comment by paulirish** [comment]
This is blocked on a fix for this: 
image

which @brendankenny is doing.

**Comment by brendankenny** [comment]
fixed with #6686

**Pull request merged**